announcements/_posts/2020-01-07-gsoc.md

```diff
@@ -3,7 +3,7 @@ title: HSF GSoC 2020 call for projects and mentors
 author: Andrei Gheata
 layout: newsletter
 symbol: glyphicon-calendar
-link: activities/gsoc.html#for-projects-and-mentors
```

Comment:
This was a valid link (was actually working), it is the checker that is wrong...

Reply:
But we can change it indeed to reduce the noise from the checker.